notanorm/mysql.py

```diff
@@ -226,10 +228,13 @@ def simplify_model(model: DbModel):
                     primary_fields = index.fields
             cols = []
             for col in tab.columns:
+                d = col._asdict()
+                if col.typ == DbType.INTEGER and not col.size:
+                    # defaults to 8 if unspecified
+                    d["size"] = 8
```

Comment:
Assuming this is the case of `CREATE TABLE xyz (col INT)`? If so, does it actually default to a 64-bit int? According to [the docs](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html), `INT` at least defaults to 32-bit. Or is the idea to diverge from "real" MySQL and use a saner default?